postgres/init.sls

```diff
@@ -8,6 +8,11 @@ include:
 {{ postgres.conf_dir }}:
   file.directory:
     - makedirs: True
+    - user: postgres
+    - group: postgres
+    - dir_mode: 700
+    - clean: True
```

Comment:
This change also causes this:

```
          ID: /etc/postgresql/9.4/main
    Function: file.directory
      Result: False
     Comment: User postgres is not available Group postgres is not available
     Started: 08:54:51.746521
    Duration: 63.534 ms
     Changes:
```

Please check your changes for idempotency before making commits and (more importantly) submitting your PR. This means they should work on a fresh box with no problems.